### PR TITLE
CRM457-1114: Add additional costs form

### PR DESCRIPTION
**Pull request by jsugarman**
## Description of change
Add additional costs form

[link to ticket](https://dsdmoj.atlassian.net/browse/CRM457-1114)

Alongside the servce costs form and travel costs forms
this is the final work, for now, on Adjustment forms.

- add form and get it working
- add javascript for updating on the page

### CODE REVIEW

File: app/view_models/prior_authority/additional_costs_with_adjustments.rb [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+        "#{I18n.t(unit_type, scope: 'prior_authority.application_details.items.per_unit_descriptions')}"
+    end
+
+    def requested_formatted_cost_total
+      NumberTo.pounds(requested_base_cost)
+    end
+
+    private
+
+    def requested_items
```
Comment (jsugarman) [inline app/view_models/prior_authority/additional_costs_with_adjustments.rb:23]: will refactor to remove these unnecessary method, inline with similar changes to servie and travel cost addjustment modules

Comment (jsugarman) [reply app/view_models/prior_authority/additional_costs_with_adjustments.rb]: refactored and rebased on top of service and travel cost forms in main

File: spec/system/prior_authority/adjust_additional_costs_spec.rb [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+    expect(page).to have_content('LAA-1234')
+    click_on 'LAA-1234'
+    expect(page).to have_current_path(prior_authority_application_path(application))
+    click_on 'Adjust quote'
+    expect(page).to have_title('Adjustments')
+  end
+
+  let(:caseworker) { create(:caseworker) }
+
+  context 'with a per item additional cost' do
```
Comment (jsugarman) [inline spec/system/prior_authority/adjust_additional_costs_spec.rb:24]:
```suggestion
  context 'with per item and per hour additional costs' do
```

**Review by dwhenry — APPROVED** [review]
LGTM - excepting the comment you raised yourself.. :)

**Pull request merged**